profiles/ding2/modules/ting/ting.entities.inc

```diff
@@ -299,6 +299,38 @@ class TingEntity extends DingEntity {
     return empty($this->reply) ? FALSE : $this->reply->getLanguage();
   }
 
+    /**
```

Comment:
Remember to lint your code.

Reply:
@rimi-itk er det egentlig ikke en forbudt fil jeg har lavet det her i?

Reply:
@sinejespersen, jo, det er det da. Godt set! Og dårligt code review af mig …